flexmeasures/data/models/planning/storage.py

```diff
@@ -86,7 +86,18 @@ def _prepare(self, skip_validation: bool = False) -> tuple:
         )
 
         # Check for required Sensor attributes
-        self.sensor.check_required_attributes([("capacity_in_mw", (float, int))])
+        storage_power_capacity_in_mw = self.flex_model.get(
+            "storage_power_capacity_in_mw",
+            self.sensor.get_attribute("capacity_in_mw", None),
+        )
+
+        if not (
+            isinstance(storage_power_capacity_in_mw, float)
+            or isinstance(storage_power_capacity_in_mw, int)
+        ):
+            raise ValueError(
+                "Storage power capacity not defined in the sensor attributes or the flex-model"
+            )
```

Comment:
We should split this into two if statements. The error message is more suitable for checking `if storage_power_capacity_in_mw is None`. The current if statement is more suitable for possibly throwing a `ValidationError` in case the variable came from a database attribute (in case the variable came from the API, it already passed validation).

Reply:
I don't think this comment has been addressed?

Reply:
It is addressed in the following line:
https://github.com/FlexMeasures/flexmeasures/blob/5dc7855f7f41b52e2c1cd752e48b8ce25fdddb41/flexmeasures/data/models/planning/storage.py#L95